api/controllers/album.js

```diff
@@ -16,20 +16,18 @@ const allAlbums = async (req, res) => {
       conn.end();
     }
   }
-};
+}
 
-exports.allAlbums = allAlbums;
-
-// function:byAlbumID
-// sends the album with the given id
-const byAlbumID = async (req, res) => {
+// // function:byAlbumID
```

Comment:
check out https://devhints.io/jsdoc it tells you a good structure to follow when commenting function.

Reply:
I will talk to you about this tomorrow.